src/migrations/20220427130214-add-sats-amount-prop-to-payments.ts

```diff
@@ -0,0 +1,99 @@
+module.exports = {
+  async up(db) {
+    const collection = db.collection("medici_transactions")
+    try {
+      const resultDebits = await collection.update(
+        { type: "payment", debit: { $gt: 0 } },
+        [{ $set: { satsAmount: { $subtract: ["$debit", "$fee"] } } }],
+        { multi: true },
+      )
+      console.log(
+        { result: resultDebits },
+        "added new props to debit medici_transactions",
+      )
+    } catch (error) {
+      console.log(
+        { result: error },
+        "Couldn't add new props to debit medici_transactions",
+      )
+    }
+
+    try {
+      const allTxns = await collection.aggregate([
+        {
+          $match: {
+            satsAmount: { $exists: true },
+          },
+        },
+        {
+          $group: {
+            _id: "$hash",
+            satsAmount: { $first: "$satsAmount" },
+            fee: { $first: "$fee" },
+            usd: { $first: "$usd" },
+            feeUsd: { $first: "$feeUsd" },
+          },
+        },
+      ])
+      for await (const { _id: hash, satsAmount, fee, usd, feeUsd } of allTxns) {
+        if (!hash) continue
+        const centsAmount = Math.round((usd - feeUsd) * 100)
+        const centsFee = Math.round(feeUsd * 100)
+
+        const resultRest = await collection.update(
+          { hash },
+          [
+            {
+              $set: {
+                satsAmount,
```

Comment:
Were we going to add ```accountingCurrency``` as a rename of ```currency```.

Reply:
Thinking to do that as part of a follow-up migration after the #1237 PR is merged, see scope bullet point here: https://github.com/GaloyMoney/galoy/pull/1269#issuecomment-1111704927